aksel.nav.no/website/pages/_app.tsx

```diff
@@ -33,6 +49,17 @@ function App({ Component, pageProps, router }: AppProps) {
 
   return (
     <>
+      <Head>
```

Comment:
All of this code can be abstracted into the `ConsentBanner`-component itself. This also includes the `clientAcceptsTracking` and `umamiTag` states. We would then only need to call `   <ConsentBanner />` here.

Reply:
Also, since state is handled in 1 component then, we shouldnt need to call `umami.track();` manually in ConsentBanner